project-base/docker/php-fpm/docker-php-consumer-entrypoint

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+default_tries_before_fail=100
+messages_to_consume_count=1000
+
+consumer_name=$1
+
+if [[ -z ${consumer_name} ]]; then
+    1>&2 echo "First argument had to be name of consumer to start"
+    exit 1
+fi
+
+if [[ -n ${CONSUMER_RUNS_BEFORE_RESTART} ]]; then
+    messages_to_consume_count=${CONSUMER_RUNS_BEFORE_RESTART}
+fi
+
+n=0
+until [[ ${n} -ge ${default_tries_before_fail} ]]; do
+    php bin/console shopsys:rabbitmq:check-availability >/dev/null 2>&1 && php bin/console >/dev/null 2>&1 && break
+    1>&2 echo "Consumer is not yet ready - sleeping"
+    n=$[$n+1]
+    sleep 5
+done
+
+if [[ ${n} = ${default_tries_before_fail} ]]; then
+    1>&2 echo "Number of maximum retries reached - failing"
+    exit 1
+fi
+
+ENVIRONMENT="$(php bin/console about | grep Environment | sed 's/Environment//;s/ //g')"
+
+if [[ ${ENVIRONMENT} = prod ]]; then
```

Comment:
The reason of this condition should be documented in a comment imo.

Reply:
I'll keep it in mind when writing documentation